city_scrapers/settings/base.py

```diff
@@ -18,7 +18,7 @@
 USER_AGENT = "City Scrapers [development mode]. Learn more and say hello at https://www.citybureau.org/city-scrapers/"  # noqa
 
 # Obey robots.txt rules
-ROBOTSTXT_OBEY = True
+ROBOTSTXT_OBEY = False
```

Comment:
Just checking: is this change to `False` needed? Did the robots text settings change?

Reply:
@SimmonsRitchie Yes this change is needed, otherwise I was getting the Forbidden error when scraping the ind_school_board site. 
https://stackoverflow.com/questions/37274835/getting-forbidden-by-robots-txt-scrapy

Reply:
Okay, thank you for the background, @haileyhoyat! So I recently came across this issue myself with a [Spokane agency](https://github.com/City-Bureau/city-scrapers-spokane/pull/12). It's generally good practice to obey `robot.txt` but I found a number of cases in the city-scrapers project where we overrode this setting. Since this is public meeting information I think we have a reasonable case for doing so.

Instead of disabling `ROBOTSTXT_OBEY` across the entire Indy project though, I recommend doing this on "as needed" basis for each spider. This makes it easy for us and contributors to see which spiders actually required this setting. Would you kindly revert `ROBOTSTXT_OBEY` back to `True` here and instead apply the setting specifically to the `ind_school_board` spider? You can see an example of how to do this in my branch [here](https://github.com/City-Bureau/city-scrapers-spokane/blob/b19d20c3442c09c6c5e77bd7a6c7f79820482d61/city_scrapers/spiders/spo_sv_city_council.py#L20).

Reply:
@SimmonsRitchie . done.